src/views/Comorbidities/Comorbidities.js

```diff
@@ -189,8 +192,10 @@ const Comorbidities = () => {
       medicacoes,
     };
 
-    if(values.gestacao_semanas) submitData.gestacao_semanas = values.gestacao_semanas;
-    if(values.puerperio_semanas) submitData.puerperio_semanas = values.puerperio_semanas;
+    if (values.gestacao_semanas)
+      submitData.gestacao_semanas = values.gestacao_semanas;
+    if (values.puerperio_semanas)
+      submitData.puerperio_semanas = values.puerperio_semanas;
```

Comment:
Se for quebrar linha, coloca umas chaves { } que desta forma funciona, mas ao passar o olho rapidamente fica meio confuso. Não vejo como uma boa pratica no clean code

Reply:
Ok :+1: